Bottom: 180d8f2c9a22f1039d8ff0813ee06dacda16eb3c
Top:    938688974ef3cfc51a54c3498a7f4d2621ae3950
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-09 15:55:29 -0500

Enable file-based secrets provider


---
diff --git a/docker/buildbot-master/buildbot/master.cfg b/docker/buildbot-master/buildbot/master.cfg
index dfbf6dec..707c1802 100644
--- a/docker/buildbot-master/buildbot/master.cfg
+++ b/docker/buildbot-master/buildbot/master.cfg
@@ -5,7 +5,7 @@
 # 'master.cfg' in your buildmaster's base directory.
 
 import os
-from buildbot.plugins import steps,util
+from buildbot.plugins import secrets,steps,util
 
 os.environ['LONG_RUN_TIMEOUT']='14400'
 
@@ -22,6 +22,9 @@ c['change_source'] = []
 # --master option)
 c['protocols'] = {"pb": {"port": int(os.getenv('BUILDBOT_WORKER_PORT',9989))}}
 
+### SECRETS MANAGEMENT
+c['secretsProviders'] = [secrets.SecretInAFile(dirname="/run/secrets")]
+
 ####### STATUS TARGETS
 
 # 'status' is a list of Status Targets. The results of each build will be
